projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -125,9 +126,12 @@ export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
 
     if (this.groupId !== undefined) {
       this.subscriptionLifeCycle.add(
-        this.chartSyncService?.getLocationChangesForGroup(this.groupId).subscribe(data => {
-          this.chart?.showCrosshair(data.locationData);
-        })
+        this.chartSyncService
+          ?.getLocationChangesForGroup(this)
```

Comment:
both group id and this should go to the sync service - no reason to filter one thing internal to the service and the other external.